src/components/assets/NativeAssetList.vue

```diff
@@ -206,7 +206,8 @@ export default defineComponent({
     const isShibuya = ref<boolean>(false);
     const isFaucet = ref<boolean>(false);
     // Memo: defined by hard-coding to avoid sending too many requests to faucet API server
-    const mainnetFaucetAmount = 0.002;
+    // Ref: https://github.com/AstarNetwork/astar-faucet-bot/blob/main/src/clients/astar.ts#L207
+    const mainnetFaucetAmount = 0.002 / 2;
```

Comment:
In stead of hard coding the value, I think it's better to have the faucet server emit the amount

Reply:
Actually, I would like to avoid requesting to faucet server as much as possible that's why I hard coded variable. Otherwise, our faucet server gets a lot of requests and the server goes down.